src/main/java/org/bytedeco/javacpp/Pointer.java

```diff
@@ -512,7 +511,13 @@ public static long parseBytes(String string, long relativeMultiple) throws Numbe
 
         String mx = System.getProperty("org.bytedeco.javacpp.mxbean", "false").toLowerCase();
         if (mx.equals("true") || mx.equals("t") || mx.equals("")) {
-            PointerBufferPoolMXBean.register();
+            try {
+                Class c = Class.forName("PointerBufferPoolMXBean");
```

Comment:
Does this need to be fully qualified?

```suggestion
                Class c = Class.forName("org.bytedeco.javacpp.tools.PointerBufferPoolMXBean");
```

Reply:
Yes, it won't work otherwise.